pkg/reconciler/volumeclaim/pvchandler.go

```diff
@@ -91,7 +92,13 @@ func getPersistentVolumeClaims(workspaceBindings []v1alpha1.WorkspaceBinding, ow
 // must be a PersistentVolumeClaim from set's VolumeClaims template.
 func GetPersistentVolumeClaimName(claim *corev1.PersistentVolumeClaim, wb v1alpha1.WorkspaceBinding, owner metav1.OwnerReference) string {
 	if claim.Name == "" {
```

Comment:
Would it make sense to use the existing name generator we have?

https://github.com/tektoncd/pipeline/blob/master/pkg/names/generate.go#L54

You can see it in use for something similar here: https://github.com/tektoncd/pipeline/blob/master/pkg/pod/creds_init.go#L75

Reply:
the problem is that its is _random_. This name must be a consistent hash, e.g. every time a workspaceName+ownerName is passed to the function, the same hash must be returned.

random name is also useful, but in this context we need a consistent hashed name.

Reply:
Ah - makes sense. Would you mind adding a comment explaining that here? It's still not clear to me why we need it to be consistent.

Reply:
@dlorenc Good point, I added an explanation in the function-comment.

Initially, a PVC is created - with workspace-name and an owner-name as input (owner-name is a TaskRun or a PipelineRun name and typically consist of a random part - using the function you referred).

Then, every TaskRun that use this workspace (within the same PipelineRun) - must refer to the exact same PVC-name. This can be handled in two ways - use a consistent name-lookup (e.g. always return the same output given two input values) or use some way to _pass_ the PVC name. 

We currently rely on returning the same PVC-name given a specific combination of ws-name and owner-name. I think this is an easier way to implement it.